### DISCUSSION TITLE
Is Laravel Event Sourcing scalable?

### DISCUSSION THREAD

**Discussion by chrispage1** [spatie/laravel-event-sourcing#275] [Q&A]
The concept of event sourcing to me seems great. It leaves you with an audit trail and gives you the benefit of being able to process data retrospectively. My only concern is, that database is really going to grow once you're handling ~300 renewals, ~400 logins and numerous other events a day and recording them as events.

What are your thoughts on this?

Chris.

**Comment by YOzaz** [comment]
Pretty much related to: https://github.com/spatie/laravel-event-sourcing/discussions/347
